jboss-kie-kieserver/added/launch/jboss-kie-kieserver.sh

```diff
@@ -180,19 +180,12 @@ function set_timer_defaults {
         if [ "x${url}" != "x" ]; then
             EJB_TIMER_XA_CONNECTION_PROPERTY_URL="${url}?pinGlobalTxToPhysicalConnection=true\&amp;${enabledTLSParameterName}=${MYSQL_ENABLED_TLS_PROTOCOLS:-TLSv1.2}"
             eval ${prefix}_URL="${url}?${enabledTLSParameterName}=${MYSQL_ENABLED_TLS_PROTOCOLS:-TLSv1.2}"
-        else
-            # the first character must be upper case
-            local paramName=${enabledTLSParameterName^}
-            EJB_TIMER_XA_CONNECTION_PROPERTY_PinGlobalTxToPhysicalConnection="true"
-            eval EJB_TIMER_XA_CONNECTION_PROPERTY_${paramName}="${MYSQL_ENABLED_TLS_PROTOCOLS:-TLSv1.2}"
```

Comment:
The TLS protocol should exists for both, you only need to remove the Pin property from the mariadb XA property.